buildkite/resource_organization_settings.go

```diff
@@ -0,0 +1,101 @@
+package buildkite
+
+import (
+  "context"
+  "fmt"
+
+  "github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+  "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceOrganizationSettings() *schema.Resource {
+  return &schema.Resource{
+    CreateContext: CreateUpdateDeleteOrganizationSettings,
+    ReadContext:   ReadOrganizationSettings,
+    UpdateContext: CreateUpdateDeleteOrganizationSettings,
+    DeleteContext: CreateUpdateDeleteOrganizationSettings,
+    Importer: &schema.ResourceImporter{
+      State: schema.ImportStatePassthrough,
+    },
+
+    Schema: map[string]*schema.Schema{
+      "id": {
+        Type:     schema.TypeString,
+        Computed: true,
+      },
+      "slug": {
+        Type:     schema.TypeString,
+        Required: true,
+      },
```

Comment:
We don't actually need this in the schema (and probably shouldn't have it duplicated). Its part of the provider configuration so is accessible in the CRUD functions